sw/device/lib/testing/test_framework/ottf_common.ld

```diff
@@ -246,5 +235,69 @@ SECTIONS {
     *(.freertos.heap)
   } > ram_main
 
+  /**
+   * Non-volatile scratch and counter areas for tests.
+   *
+   * These sections are meant to be used by tests to store data that should
+   * persist across resets. Since `opentitantool` assumes that the signed
+   * region of an image extends until the end, these sections are placed after
+   * all other sections and are NOLOAD so that they don't appear in the binary
+   * and therefore don't end up being covered by the signature.
+   *
+   * Note that these sections are placed in `eflash` instead of `ottf_flash` so
+   * that they don't interfere with tests that use address translation.
+   */
+   _non_volatile_scratch_size = 20480;
```

Comment:
did you mean 20480? or did you mean to write 2048?

Reply:
The lines below seem to indicate that each counter is 2048 bytes, and we have 3 of them, suggesting that the correct size should be 2048 * 3? That still seems quite high. Is there documentation to check these values?

Reply:
@luismarques just to clarify: the scratch regions and counters are all separate sections. With this change we have 3 separate counters that can count up to 256 safely and a scratch region that can store up to 20 KiB. All of these are placed at the end of a slot, i.e. one half of a flash which is 512 KiB.